spec/models/vm/retirement_management_spec.rb

```diff
@@ -44,9 +49,7 @@
 
     context "without user" do
       before do
-        # system_context_retirement relies on the presence of a user with this userid
-        FactoryBot.create(:user, :userid => 'admin', :role => 'super_administrator')
-        user.destroy
+        User.first.destroy
```

Comment:
What `User` record are you actually removing here.  The `.first` call makes me think this could lead to sporadic test failures.  Is there a better way to handle this?